src/pages/AddGamePage.js

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components/macro'
+import Search from '../components/Search'
+
+//const api_key = process.env.REACT_APP_API_KEY;
+
+export default function AddGamePage({ onSubmit, games }) {
+  return (
+    <Container>
+      <Search onSubmit={onSubmit} />
+      <List>
+        {games.map(game => (
+          <Li backgroundImage={game.background_image}>{game.name}</Li>
+        ))}
+      </List>
+    </Container>
+  )
+}
+
+const Container = styled.section`
+  display: grid;
+  justify-items: center;
+
+  Button {
```

Comment:
Would be better to restyle the `Button` component in the `Search.js`. With this being here, the styles and the corresponding code are separated and we want to avoid that.
```js
const FormButton = styled(Button)`...`
```

Reply:
I don't know how to style the button in this file. It's not in the code because it's imported. I just removed line 23 from the styled component in this file. Hope it's fine like that. :)